adafruit_pn532/spi.py

```diff
@@ -51,12 +51,7 @@
 def reverse_bit(num):
     """Turn an LSB byte to an MSB byte, and vice versa. Used for SPI as
     it is LSB for the PN532, but 99% of SPI implementations are MSB only!"""
-    result = 0
-    for _ in range(8):
-        result <<= 1
-        result += (num & 1)
-        num >>= 1
-    return result
+    return int('{:08b}'.format(num)[::-1], 2)
```

Comment:
This currently isn't supported in CircuitPython. Only slices with step=1. That's probably why this was being done manually.
```python
NotImplementedError: only slices with step=1 (aka None) are supported
```

Reply:
Oh, OK not a problem.

Reply:
Is there anyway I can check these check these differences in the future? I ran it in the Mu and it never pointed that out.